scripts/assets/test_file_exemptions.textproto

```diff
@@ -504,8 +504,8 @@ exempted_file_path: "app/src/sharedTest/java/org/oppia/android/app/utility/FontS
 exempted_file_path: "app/src/sharedTest/java/org/oppia/android/app/utility/OrientationChangeAction.kt"
 exempted_file_path: "app/src/sharedTest/java/org/oppia/android/app/utility/ProgressMatcher.kt"
 exempted_file_path: "app/src/sharedTest/java/org/oppia/android/app/utility/TabMatcher.kt"
-exempted_file_path: "app/src/test/java/org/oppia/android/app/testing/ProfileProgressSpanCount.kt"
-exempted_file_path: "app/src/test/java/org/oppia/android/app/testing/player/split/PlayerSplitScreenTesting.kt"
+exempted_file_path: "app/src/test/java/org/oppia/android/app/testing/ProfileProgressSpanCountTest.kt"
+exempted_file_path: "app/src/test/java/org/oppia/android/app/testing/player/split/PlayerSplitScreenTest.kt"
```

Comment:
I think these two lines can be removed now. Can you try this change and see if CI passes?

Reply:
@rt4914  All tests passed. PTAL.